test/dotnet-test.Tests/GivenThatWeWantToRunTestsInTheConsole.cs

```diff
@@ -95,6 +95,36 @@ public void It_skips_build_when_the_no_build_flag_is_passed()
             result = testCommand.Execute($"{_projectFilePath} -o {_defaultOutputPath} --no-build");
             result.Should().Pass();
         }
+        
+        [Theory]
+        [InlineData("--output", "Missing value for option 'output'")]
+        [InlineData("--configuration", "Missing value for option 'configuration'")]
+        [InlineData("--runtime", "Missing value for option 'runtime'")]
+        [InlineData("--build-base-path", "Missing value for option 'build-base-path'")]
+        public void It_fails_correctly_with_unspecified_arguments_with_long_form(string arguments, string expectedStdErr)
+        {
+            new DotnetTestCommand()
+                .ExecuteWithCapturedOutput($"{_projectFilePath} {arguments}")
+                .Should()
+                .Fail()
+                .And
+                .HaveStdErrContaining(expectedStdErr);
```

Comment:
Instead of having "Missing value for option" duplicated for each test, it could be easier to have a method:

```C#
private static string GetMissingOptionMessage(string argument)
{
  return $"Missing value for option '{argument}'";
}
```

And then all the tests call it:

```
.HaveStdErrContaining(GetMissingOptionMessage(arguments.Substring(1 or 2)));  //depending on long or short
```

Then you wouldn't even need `expectedStdErr` parameter either.